management-account/terraform/cost-and-usage-reports.tf

```diff
@@ -65,7 +65,7 @@ resource "aws_cur_report_definition" "moj_cur_report" {
   report_versioning          = "OVERWRITE_REPORT"
 
   # S3 configuration
-  s3_bucket = module.cur_reports_v2_hourly_s3_bucket.bucket_name
+  s3_bucket = module.cur_reports_v2_hourly_s3_bucket.bucket.bucket
```

Comment:
This does not look like it's resolving

Reply:
The error clearly states, `bucket.bucket` is an object with a number of attributes, where you want a string that carries a value of the bucket name:
```╷
│ Error: Incorrect attribute value type
│ 
│   on cost-and-usage-reports.tf line 68, in resource "aws_cur_report_definition" "moj_cur_report":
│   68:   s3_bucket = module.cur_reports_v2_hourly_s3_bucket.bucket
│     ├────────────────
│     │ module.cur_reports_v2_hourly_s3_bucket.bucket is object with 28 attributes
│ 
│ Inappropriate value for attribute "s3_bucket": string required.
```

Reply:
sorry, missed the double bucket name, this should be fine